orchestra/migrations/0018_auto_20151007_1451.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+def update_projects(apps, schema_editor):
+    Project = apps.get_model('orchestra', 'Project')
+    WorkflowVersion = apps.get_model('orchestra', 'WorkflowVersion')
+
+    for project in Project.objects.all():
+        project.workflow_version = WorkflowVersion.objects.get(
+            slug=project.workflow_slug)
+        project.save()
+
+
+def update_tasks(apps, schema_editor):
+    Task = apps.get_model('orchestra', 'Task')
+    WorkflowStep = apps.get_model('orchestra', 'WorkflowStep')
+
+    for task in Task.objects.all():
+        task.step = WorkflowStep.objects.get(slug=task.step_slug)
+        task.save()
+
+
+def migrate_worker_certs(apps, schema_editor):
+    Certification = apps.get_model('orchestra', 'Certification')
```

Comment:
This handles the fact that certifications didn't used to be scoped to a workflow. This function takes existing worker_certifications and creates a copy for every new workflow that defines a certifications with the same slug.